pennylane/devices/qutrit_mixed/utils.py

```diff
@@ -68,6 +69,37 @@ def get_einsum_mapping(
     )
 
 
+def resquare_state(state, num_wires):
```

Comment:
Maybe rename as some variant of `reshape_state`? Since `square` also has the mathematical connotation of squaring.

Reply:
changed to `reshape_state_as_matrix`